lisa_commands/commands.py

```diff
@@ -252,14 +252,53 @@ def image_me(query, _slack_event):
         return "From Google: " + request.google_image_search(query)
     return "From Bing: " + request.bing_image_search(query)
 
-def kill_me(_query, _slack_event):
+def kill_me(_query, slack_event):
     """
     query - query str (unused for this function)
-    slack_event - A dict of slack event information (unused for this function)
+    slack_event - A dict of slack event information
 
-    Returns a link to a gif
+    Returns nothing
     """
-    return request.gify_search('gifs', 'kill me')
+
+    # kick user from channel
+    data = {
+        'channel': slack_event["channel"],
+        'user': slack_event['user'],
+    }
+
+    response = json.loads(request.submit_slack_request(data, 'channels.kick', 'USER'))
+    if response['ok']:
+        text = random.choice([
+            'By order of the SDF Armed Forces you have been killed',
+            'Ask not for whom the bell tolls. It tolls for thee ~ John Donne',
+            "We all have been one life to live. Well you do. I'm a bot, and I have backups",
```

Comment:
nit: 'we all have `but` one life to live' (Unless I'm remembering the quote wrong)

Reply:
No, I think you're right. I'd back this change.